ibazel/command/default_command.go

```diff
@@ -64,9 +65,9 @@ func (c *defaultCommand) Start() {
 
 	var err error
 	if err = c.cmd.Start(); err != nil {
-		fmt.Printf("Error starting process: %v\n", err)
+		fmt.Fprintf(os.Stderr, "Error starting process: %v\n", err)
 	}
-	fmt.Printf("Starting...")
+	fmt.Fprintf(os.Stderr, "Starting...")
```

Comment:
Stdout?

Reply:
since stdout/err are combined when displayed on screen, my plan is to reserve stderr for the communication to the user from `ibazel` leaving the stdout pipe free for your program to output stuff. This is in the same tradition as `bazel` proper which has this output (when you pipe stdout to /dev/null and totally ignore it)

```
achew22@lappy:~/.../bazel-watcher (e2e)$ bazel build //... > /dev/null
.........
Loading: 
Loading: 0 packages loaded
Analyzing: 19 targets (9 packages loaded)
Analyzing: 19 targets (16 packages loaded)
WARNING: <snip>/WORKSPACE:1: Workspace name in <snip>/WORKSPACE (@io_bazel_rules_go) does not match the name given in the repository's definition (@com_github_bazelbuild_rules_go); this will cause a build error in future versions
Analyzing: 19 targets (32 packages loaded)
INFO: Analysed 19 targets (37 packages loaded).
INFO: Found 19 targets...
[1 / 6] BazelWorkspaceStatusAction stable-status.txt
[170 / 180] no action running
INFO: Elapsed time: 6.782s, Critical Path: 0.24s
INFO: Build completed successfully, 1 total action
```
So all the things that aren't explicitly important to your build go in the stderr pipe.

It turns out this functionality is really useful because it lets you pipe the output of your program to a file and still see build progress and stuff.